### ISSUE TITLE
[High]: Chrome+JAWS: Screen reader: Role is not defined for control.

### ISSUE THREAD

**Issue by SumitDiyora** [Expensify/App#8393]
## Action Performed:
1. Using Chrome+JAWS, open URL: staging.new.expensify.com
2. Navigate to Settings > Workspace > Manage members > Invite
3. Navigate to the 'Invite' control using the screen reader.
4. Observe the screen reader announcement.

## Expected Result:
Role should be defined as a 'button' for the "Invite" control.

## Actual Result:
Role is not defined for the "Invite" control. This made it difficult for screen reader users to access the functionality associated with it.

**Other Occurrences:**
>Similar type of issue repro on PRs https://github.com/Expensify/App/pull/8143, #8017, #8039, #8019, #8196, #8452, #8448, #7362, #8256, #8584, #8694,#8969, #8878, #9144,
>Same issue repro on PR #8494 -Role is not defined for new group.
>Same issue repro on PR #9369, #9056 -Role is not defined for 'Save' button.
>Same issue repro on PR #9144  -Role is not defined for 'Create group' button.
>Same issue repro on PR #9351   -Role is not defined for 'Confirm' button.
>Same issue repro on PR #9400   -Role is not defined for 'Close Account' button.
>Same issue repro on PR #9348    -Role is not defined for 'Add a payment method' button.

## Workaround:
Yes

## Area issue was found in:
Invite new members

## Failed WCAG checkpoints
4.1.2

## User impact:
Screen reader users will not be made aware that these elements are buttons.

## Suggested resolution:
Ensure that all the page functionality is available for all users irrespective of the device being used.
Apply the following changes:
- Specify role="button" for the 'div' element containing the "Invite" control.

## Platform:
- Web
- iOS
- Android
- Desktop App
- Mobile Web

**Version Number:** 1.1.49-0
**Reproducible in staging?:** Yes
**Reproducible in production?:** Yes
**Reference link:** https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role
**Issue reported by:** Sumit

![#7920_Role and label not defined for Invite control](https://user-images.githubusercontent.com/98817464/161062728-a67da0dd-628b-4c9a-a50b-30c100bf7036.png)

**Comment by SumitDiyora** [comment]
> ## Action Performed:
> 1. Using Chrome+JAWS, open URL: staging.new.expensify.com
> 2. Navigate to Settings > Workspace > Manage members > Invite
> 3. Navigate to the 'Invite' control using the screen reader.
> 4. Observe the screen reader announcement.
> 
> ## Expected Result:
> Role should be defined as a 'button' for the "Invite" control.
> 
> ## Actual Result:
> Role is not defined for the "Invite" control. This made it difficult for screen reader users to access the functionality associated with it.
> 
> ## Workaround:
> Yes
> 
> ## Area issue was found in:
> Invite new members
> 
> ## Failed WCAG checkpoints
> 4.1.2
> 
> ## User impact:
> Screen reader users will not be made aware that these elements are buttons.
> 
> ## Suggested resolution:
> Ensure that all the page functionality is available for all users irrespective of the device being used. Apply the following changes:
> 
> * Specify role="button" for the 'div' element containing the "Invite" control.
> 
> ## Platform:
> * Web
> * iOS
> * Android
> * Desktop App
> * Mobile Web
> 
> **Version Number:** 1.1.49-0 **Reproducible in staging?:** Yes **Reproducible in production?:** Yes **Reference link:** https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role **Issue reported by:** Sumit
> 
> ![#7920_Role and label not defined for Invite control](https://user-images.githubusercontent.com/98817464/161062728-a67da0dd-628b-4c9a-a50b-30c100bf7036.png)
